### DISCUSSION TITLE
New user registration: Something isn't quite right yet! Please review the error below

### DISCUSSION THREAD

**Discussion by ralfr** [mastodon/mastodon#22039] [Q&A]
My instance has been open for registrations and users have registered successfully. All of a sudden and with no changes whatsoever on the backend, users start reporting that the web UI returns

`Something isn't quite right yet! Please review the error below`

when submitting the new user registration form.

I investigated the following:

- No errors in nginx logs.
- `journalctl` doesn't return any errors for `mastodon-web`.
- Postgres is all green.
- All Sidekiq processes are stable and all queues are down to zero.
- No JavaScript errors in the JS Console on the frontend.
- SMTP is working. Other activities requiring SMTP (password resets, etc.) work.
- When switching registration to Waitlist, registration works. **Switching back to open registration and the bug is gone.**

Do you guys have any ideas how to further troubleshoot?

Mastodon 4.0.2, non-docker

**Comment by ziIIas** [comment]
did u ever fix it
